src/bikeracks/templates/scripts.js

```diff
@@ -366,13 +360,13 @@ BikeMap.prototype.createMarker = function(state) {
     else {
         // we didn't find a marker, make a new one
         marker = L.marker([state.latitude, state.longitude], {uniqueId: state.rack_id}); 
-        
         // add marker to allRacks feature group and its feature group based on status
         this.allRacks.addLayer(marker);
-        if (state.status === "approved") {
+        // true implies that this rack is approved
```

Comment:
Now that you have a function with a useful name this comment is no longer needed.